### ISSUE TITLE
[FE] 10-10EZ Short-form: Correct Insurance Information Legend > [Markup and meta data] Visual labels aren't included in the accessible label. (09.05.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#40487]
#### VFS team name
10-10 Team

#### VFS product name
Health Care Application 10-10EZ

#### VFS feature name
10-10EZ Short Form (Auto-qualifiers)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Visual labels aren't included in the accessible label.

### Issue Details
On Step 2 of 3 [disability rating >50%]: Insurance Information, when a user has health insurance coverage and is adding an insurance policy, the prompt "Provide either your insurance policy number or group code" isn't programmatically associated with the related fields. It's styled like a `<legend>` but isn't actually a legend, which means a screen reader user tabbing through the form fields may miss it entirely --- including the indication that this information is required.

### VA.gov Experience Standard
[Category Number 09, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.5.3_A

---

### Platform Recommendation
Group the Policy number and Group code fields together in a `<fieldset>`, and make the "Provide either your..." prompt the `<legend>` element for that fieldset.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
> On Step 2 of 3 [disability rating >50%]: Insurance Information, when a user has health insurance coverage and is adding an insurance policy, the prompt "Provide either your insurance policy number or group code" isn't programmatically associated with the related fields. It's styled like a <legend> but isn't actually a legend, which means a screen reader user tabbing through the form fields may miss it entirely --- including the indication that this information is required.

@briandeconinck 
This functionality was tested with @AngelaFowler82 and there was no feedback at the time, it seemed fine.   

@AngelaFowler82 - can you please review this finding also, and let us know your thoughts?

**Comment by AngelaFowler82** [comment]
Can you send a direct link to the component as well as the user account we're using? Huge thanks.

**Comment by hdjustice** [comment]
@AngelaFowler82 - The work was in a Review Instance, rather than Staging environment.  That Review instance has expired, so I cannot provide you with the link at this time.  I will see if I can get the Review Instance reactivated.

**Comment by hdjustice** [comment]
Hi @AngelaFowler82 - I talked with Lisa Zapson and she says that when she makes changes to the code, she has to generate a new Review Instance, as she is working on some of the other feedback we received.  Would you be able to identify a day that you could review this component?  I will then get the appropriate link from Lisa and provide you the information you need to review this item.  Thank you!

Reminder on component in question: 
On Step 2 of 3 [disability rating >50%]: Insurance Information, when a user has health insurance coverage and is adding an insurance policy, the prompt "Provide either your insurance policy number or group code" isn't programmatically associated with the related fields. It's styled like a legend but isn't actually a legend, which means a screen reader user tabbing through the form fields may miss it entirely, including the indication that this information is required.

**Comment by hdjustice** [comment]
Hello @AngelaFowler82 
I did confirm that the Insurance information page referenced in this staging review finding ticket is in Staging environment rather than the Review Instance.  So you can actually access it without requiring SOCKS.
Per the finding in this ticket, the prompt "Provide either your insurance policy number or group code" isn't programmatically associated with the related fields. It's styled like a legend but isn't actually a legend, which means a screen reader user tabbing through the form fields may miss it entirely, including the indication that this information is required.

This is the [direct link to the insurance information page](https://staging.va.gov/health-care/apply/application/insurance-information/general) in the staging environment.  After the policy holder name field, there is a line of text that states "Provide either your insurance policy number or group code". Below this line, there is an accordion dropdown that states "I have TRICARE. What's my policy number?"

Let me know if you would like to chat through this.  I am happy to schedule some time.

**Comment by AngelaFowler82** [comment]
I have tested this with Jawsand NVDA end it works in both cases. @ @
